### PR TITLE
Close session in worker_webhook function

### PR DESCRIPTION
**Pull request by kkaarreell**
Resolves https://github.com/keylime/keylime/issues/1455

### CODE REVIEW

**Comment by kkaarreell** [comment]
There is a Packit service outage and therefore Packit tests have not been started yet.

**Review by stefanberger — APPROVED** [review]

**Review by sergio-correia — APPROVED** [review]

File: keylime/revocation_notifier.py [hunk 1]
```diff
@@ -159,6 +159,8 @@ def worker_webhook(tosend: Dict[str, Any], url: str) -> None:
 
             time.sleep(next_retry)
 
+        session.close()
```
Comment (aplanas) [inline keylime/revocation_notifier.py:162]: I think that a bit more correct fix is to use the context manager: https://requests.readthedocs.io/en/latest/user/advanced/#session-objects

Comment (kkaarreell) [reply keylime/revocation_notifier.py]: Updated.

**Review by stefanberger — APPROVED** [review]

**Review by sergio-correia — APPROVED** [review]

**Pull request merged**